test/core/util/byteSerialization/mock.ts

```diff
@@ -0,0 +1,179 @@
+import { Transaction, TransactionType } from 'shared/model/transaction';
+import { BlockModel } from 'shared/model/block';
+
+export const recipientAddress = BigInt('15984232642719285606');
+export const senderAddress = BigInt('4995063339468361088');
+
+/* tslint:disable:no-magic-numbers */
+
+export const amount = 10000000000;
+
+export const transactionRegister = new Transaction({
+    id: '85519724a269996537c1c3533f912016c34b782678f96a0d1a35459b453611de',
+    type: TransactionType.REGISTER,
+    createdAt: Math.floor(Date.now() / 1000),
```

Comment:
It's invalid time, we should use epoch time

Reply:
getEpochTime is private